metrics_sdk/lib/opentelemetry/sdk/metrics/export/console_metric_exporter.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module SDK
+    module Metrics
+      module Export
+        class ConsoleMetricExporter
+          PREFERRED_TEMPORALITY = 'delta'
```

Comment:
Since we're using `frozen_string_literal`, is there any benefit to the named constant?